runloop.c

```diff
@@ -3615,9 +3680,51 @@ bool runloop_init_libretro_symbols(
 
                if (string_is_empty(path))
                {
-                  RARCH_ERR("[Core]: Frontend is built for dynamic libretro cores, but "
+                  const char* content_path = path_get(RARCH_PATH_CONTENT);
+                  if (!string_is_empty(content_path))
+                  {
+                     bool core_loaded = auto_load_core(content_path);
+                     if (!core_loaded) {
+                        struct menu_state* menu_state = menu_state_get_ptr();
+
+                        CORE_SYMBOLS(SYMBOL_DUMMY);
+                        runloop_set_current_core_type(CORE_TYPE_DUMMY, false);
+#ifdef HAVE_MENU
+                        /* this is a hack, we should be deferring the core list until after we have loaded menu */
+                        /* TODO: ask how to unhackify it */
```

Comment:
Hmmm, good point. I haven't tested this without the menu being available yet. Do you have any ideas on how we could "do it right"?

Reply:
I would think automatically picking the first in the list would be pretty much the only way to go.

Reply:
In other projects with similar issues I've printed out a list of options and asked the user to enter a number.

Reply:
Hi @zig-for will it be possible to rebase so that you can get the two fixes for the option `--disable-menu` ? 

Thank you.